plugins/mumble_plugin_linux.h

```diff
@@ -169,17 +200,16 @@ static inline bool peekProc(PTR_TYPE base, void *dest, size_t len) {
 	return (nread != -1 && static_cast<size_t>(nread) == in.iov_len);
 }
 
-template<class T>
-T peekProc(PTR_TYPE base) {
-	T v = 0;
-	if (!peekProc(base, reinterpret_cast<T *>(&v), sizeof(T))) {
+procptr_t peekProcPtr(procptr_t base) {
```

Comment:
Should document that this relies on initialize.

Reply:
In a new pull request, as this one is focused on the automatic pointer size, not on cleaning up.

Reply:
The dependency was introduced by making it rely in the 64bit check so imho documenting that belongs into the PR

Reply:
No, every `peekProc()` function relies on `initialize()`.